enterprise/pulsar.py

```diff
@@ -384,17 +388,22 @@ def _get_radec(self, model):
 
     def _get_ssb_lsec(self, toas, obs_planet):
         """Get the planet to SSB vector in lightseconds from Pint table"""
+        if obs_planet not in toas.table.colnames:
+            err_msg = f"{obs_planet} is not in toas.table.colnames. Either "
+            err_msg += "`planet` flag is not True  in `toas` or further Pint "
+            err_msg += "development to add additional planets is needed."
+            raise ValueError(err_msg)
         vec = toas.table[obs_planet] + toas.table["ssb_obs_pos"]
         return (vec / const.c).to("s").value
 
     def _get_planetssb(self, toas, model):
         planetssb = None
         if self.planets:
             planetssb = np.zeros((len(self._toas), 9, 6))
-            # planetssb[:, 0, :] = self.t2pulsar.mercury_ssb
-            # planetssb[:, 1, :] = self.t2pulsar.venus_ssb
+            # planetssb[:, 0, :3] = self._get_ssb_lsec(toas, "obs_mercury_pos")
```

Comment:
So we're leaving some of the orbits blank in planetssb?

Reply:
Some of the planet vectors are not available in Pint yet.  I put in a (hopefully) informative exception to that effect. I suppose I could clean up the code, but it seemed useful to have the others there. BayesEphem pulls by index, so we do need the zeros to space things out correctly. 

That said, the issue we had from the start was that `pos_t` was set to zero very quietly. Maybe I should add a logging message that mentions which planets are zero?

Reply:
How about we:

- add a comment to the BayesEphem code where planetssb is used, that planets X, Y, and Z are unavailable pending PINT enhancements...  
- make the empty planets NaN, not zero.